csaf_2.1/prose/edit/src/conformance.md

```diff
@@ -520,6 +520,9 @@ Secondly, the program fulfills the following for all items of:
   > This is a common case for CSAF 2.0 documents labeled as TLP:RED but actually intended to be TLP:AMBER+STRICT.
 
   If no TLP label was given, the CSAF 2.0 to CSAF 2.1 converter SHOULD assign `TLP:CLEAR` and output a warning that the default TLP has been set.
+* `/document/publisher/category`: If the value is `other`, the CSAF 2.0 to CSAF 2.1 converter SHOULD output a warning that some parties have
+  been separated into the new value `multiplier`. An option to suppress this warning MUST exist. In addition, an option SHOULD be provided to
```

Comment:
Question: I try to figure out what it means to separate multiple parties into the new value `multiplier`. Maybe an example would help?

Reply:
IMHO, those three parties are doing more or less the same:
- **forwarder**: just replace the publisher details but basically keep the rest of the original CSAF file
- **republisher**: forwarder that also might convert a "non-CSAF document" of a 3rd-party into a CSAF document
- **multiplier**: republisher that also adds additional context for a specific target group and might publish multiple versions for different audiences

IMHO, it is hard in practice to differentiate them as the borders are fluent:
- Sometimes, a **multiplier** might take a CSAF document as input and has no additional insights (e.g. the original CSAF document already covers the case very well), so he would technically speaking be a **forwarder**.
-  Or, a **multiplier** might take a non-CSAF document as input and has no additional insights (e.g. the original document already covers the case very well), so he would technically speaking be a **republisher**.

Determining that automatically, whether someone changed something (especially when converting from human-readable documents) or whether just the fact that it gets republished by a certain party is already adding context seems to be very complex. Moreover, this might change with every CSAF document (or even between versions of the same CSAF document). Therefore, I do not see a benefit in making 3 separate roles. (Also, for the PMD one has to decide for one category anyway.)

@sthagen Does that clarify the reasoning?

Reply:
@tschmidtb51 Thank you for providing the explanation of the concept and rationale.
That all reads reasonable to me.

My problem with the sentence is not specific to CSAF roles, sorry for not being clear in the first place.

Question put differently: Should a converter state that for the category source value `other` has been mapped in the target to `multiplier`? 

Because separating many into one reads wrong (the action/verb).

Reply:
@sthagen Thanks for pointing that out. This has been addressed by rephrasing in [4049a91](https://github.com/oasis-tcs/csaf/pull/747/commits/4049a916526ceb3108cd59f1ea71c8d8db8cd7b0)